src/viz-parole-revocation/VizParoleRevocationContainer.js

```diff
@@ -0,0 +1,92 @@
+import PropTypes from "prop-types";
+import { ascending, group } from "d3-array";
+import React, { useState, useEffect } from "react";
+import VizParoleRevocation from "./VizParoleRevocation";
+import {
+  DIMENSION_KEYS,
+  TOTAL_KEY,
+  VIOLATION_TYPES,
+  THEME,
+} from "../constants";
+
+// TODO: may not be necessary if source files change
```

Comment:
Likely to still be required. Exports out of BigQuery will generally tend to cast integer values to strings by default when exporting as json.

Reply:
yeah i was talking to julia about this and it seemed like that was probably going to be the case ... this was mostly a note to self, I think i should take it out